pkg/cmd/server/origin/run_components.go

```diff
@@ -265,9 +268,15 @@ func (c *MasterConfig) RunDeploymentController() {
 	if err != nil {
 		glog.Fatalf("Unable to initialize deployment controller: %v", err)
 	}
+
+	port, err := kubeserver.GetKubernetesMasterPort(&c.Options)
+	if err != nil {
+		glog.Fatal(err)
+	}
+	master := fmt.Sprintf("https://kubernetes.default.svc.%s:%d", c.Options.DNSConfig.Domain, port)
```

Comment:
Is this a tls signing thing where you don't want to sign the service ip for
the master?

On Oct 28, 2015, at 4:49 PM, Jordan Liggitt <notifications@github.com>
wrote:

In pkg/cmd/server/origin/run_components.go
<https://github.com/openshift/origin/pull/5435#discussion_r43315755>:

> @@ -265,9 +268,15 @@ func (c *MasterConfig) RunDeploymentController() {
>  	if err != nil {
>  		glog.Fatalf("Unable to initialize deployment controller: %v", err)
>  	}
> +
> +	port, err := kubeserver.GetKubernetesMasterPort(&c.Options)
> +	if err != nil {
> +		glog.Fatal(err)
> +	}
> +	master := fmt.Sprintf("https://kubernetes.default.svc.%s:%d", c.Options.DNSConfig.Domain, port)

@smarterclayton <https://github.com/smarterclayton> opinions on this? I
still don't see why the deployer would have problems using the configured
master address

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/5435/files#r43315755>.